src/services/TestDataService.ts

```diff
@@ -633,6 +634,27 @@ export class TestDataService {
         return rule;
     }
 
+    /**
+     * Creates a new basic landing page layout.
+     *
+     * @param cmsPageType - The type of the cms page layout (page/landingpage/product_detail...).
```

Comment:
CMS page types can be landingpage, page, product_list, product_detail 

please add the missing one in the doc.